test/fixtures/output/flink/artifact/create-python.golden

```diff
@@ -0,0 +1,6 @@
++----------------+---------------------+
+| Name           | my-custom-plugin    |
+| Plugin ID      | ccp-123456          |
+| Version ID     | ver-123456          |
+| Content Format | PYTHON              |
```

Comment:
content format is either JAR or ZIP

Reply:
I see so in case of `runtime-language` as `python` it will be zip ?

Reply:
`content format` is independent of `runtime language`. we will all combination. if anything changes, this can be easily update on server side without changing CLI.